packages/landing-site/src/layouts/base.astro

```diff
@@ -19,6 +18,7 @@ const { title } = Astro.props
     <title>{title}</title>
 		<meta name="robots" content="noindex">
 		<meta name="viewport" content="width=device-width, initial-scale=1">
+		<SocialMetaTags title={title} siteName="landing" />
```

Comment:
The landing page can be shared as well. The only issue left to resolve is an appropriate share image for the landing page like the framework sites.

Reply:
Is it this new one that Dustin just shared?
https://www.figma.com/file/kbSN3VYmIzdjQEx5mbBTSs/Framework-Dev?node-id=2822%3A128282